src/main/java/com/ixortalk/organization/api/service/UserService.java

```diff
@@ -23,37 +23,28 @@
  */
 package com.ixortalk.organization.api.service;
 
-import com.ixortalk.autoconfigure.oauth2.auth0.mgmt.api.Auth0Roles;
-import com.ixortalk.organization.api.domain.Organization;
 import com.ixortalk.organization.api.domain.User;
 import com.ixortalk.organization.api.rest.UserRestResource;
 
 import javax.inject.Inject;
 import javax.inject.Named;
 import javax.transaction.Transactional;
 
-import static com.google.common.collect.Sets.newHashSet;
-
 @Named
 public class UserService {
 
     @Inject
     private UserRestResource userRestResource;
 
-    @Inject
-    private Auth0Roles auth0Roles;
-
     @Transactional
```

Comment:
`@Transactional` seems not required on these methods since they're call from a controller which is transactional already?  Seems like it was there already but has been missed back then :-)